frontend/src/scenes/billing/billingLogic.ts

```diff
@@ -67,6 +67,7 @@ export const billingLogic = kea<billingLogicType>([
         registerInstrumentationProps: true,
         setRedirectPath: true,
         setIsOnboarding: true,
+        setPerksCTADismissed: (perksCTADismissed: boolean) => ({ perksCTADismissed }),
```

Comment:
I don't think we need this action, can just keep it there until they sign up for annual.